### PR TITLE
Update cluster definitions (onOff, levelControl, electricalMeasurement) and initial release of metering

### PR DESCRIPTION
**Pull request by TedTolboom**

### CODE REVIEW

File: lib/clusters/metering.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  powerFactor: { id: 6, type: ZCLDataTypes.int8 },
+  // CurrentMaxDemandDeliveredTime: { id: 8, type: ZCLDataTypes.UTC },
+  // CurrentMaxDemandReceivedTime: { id: 9, type: ZCLDataTypes.UTC },
+  profileIntervalPeriod: { id: 15, type: ZCLDataTypes.enum8 },
+  status: { id: 512, type: ZCLDataTypes.map8 },
+  unitofMeasure: { id: 768, type: ZCLDataTypes.enum8 },
+  multiplier: { id: 769, type: ZCLDataTypes.uint24 },
+  divisor: { id: 770, type: ZCLDataTypes.uint24 },
+  summationFormatting: { id: 771, type: ZCLDataTypes.map8 },
+  demandFormatting: { id: 771, type: ZCLDataTypes.map8 },
```
Comment (RobinBol) [inline lib/clusters/metering.js:21]: These ids can't be right.

File: lib/clusters/metering.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  meteringDeviceType: { id: 773, type: ZCLDataTypes.map8 },
```
Comment (RobinBol) [inline lib/clusters/metering.js:22]: Are you sure about the ids in this cluster? `meteringDeviceType` should have id `0x0306 === 774`. Please check all the ids very carefully, once this is merged it is very difficult to spot bugs here.

**Comment by RobinBol** [comment]
By the way, let's keep PRs smaller in the future, scoped per cluster for example. That way we can review and merge these things faster :)

File: lib/clusters/metering.js (hunk 2 above)
Comment (TedTolboom) [reply lib/clusters/metering.js:22]: I'll re-check them; I would suspect that a commit was not pushed to the PR

File: lib/clusters/metering.js (hunk 1 above)
Comment (TedTolboom) [reply lib/clusters/metering.js]: removed

File: lib/clusters/metering.js (hunk 2 above)
Comment (TedTolboom) [reply lib/clusters/metering.js]: removed

**Comment by TedTolboom** [comment]
Resolved all issues with above commits... and separated the PR for making the cluster definition complete

**Comment by RobinBol** [comment]
Great, thanks for your PR!

**Pull request merged**